Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,24 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner.js'
+import CardContent from './CardContent.js'
+
+
+
+const CardContainer = props => {
+    return (
+        <>
+        <div class="card-container">
+        <div className="top-card">
```

Comment:
There are a little too many divs here. It will be hard to keep track of the styling on the widths. I would get rid of them. The components themselves have an outer container that you can use to style with.